packages/preview/fyrst-ru-labreport/0.1.0/template/main.typ

```diff
@@ -0,0 +1,239 @@
+#import "@preview/fyrst-ru-labreport:0.1.0":*
+
+#import "@preview/unify:0.7.0":*
+
+#import "@preview/cetz:0.3.1"
+#import "@preview/cetz:0.3.1":*
+
+#import "@preview/cetz-plot:0.1.0"
+
+#show: project.with(
+  title: [Paper Title],
+  course-name: [Course Name],
+  course-abrev: [COURSE],
+  organization: [Reykjavík University],
+  logo:image("Graphics/ru-logo.svg",width:40%),
+  authors:(
+    (
+      name:"Jack Jones",
+      email:"jackj@org.com",
+      phone:none
+    ),
+    (
+      name:"Jill Jones",
+      email:"jillj@org.com",
+      phone:none
+    ),
+    (
+      name:"Jane Jones",
+      email:"janej@org.com",
+      phone:none
+    ),
+    (
+      name:"Joe Jones",
+      email:"joej@org.com"
+    ),
+    (
+      name:"Juan Jones",
+      email:"juanj@org.com"
+    )
+  ),
+  author-columns:3,
+  supervisors:(
+    (
+      name:"Big Man",
+      title:"Incharge",
+      email:"important@org.com",
+      phone:"+001-123-4567"
+    ),
+    (
+      name:"Less Man",
+      title:"Middle",
+      email:"lessm@org.com",
+      phone:none
+    ),
+  ),
+  supervisors-columns:2,
+  bibliography:bibliography("bibliography.bib"),
+  paper-size:"a4",
+  lang:"is",
+)
+
+
+= Inngangur

```

Comment:
Some of your line endings have `\r` / `^M`, it would be nice if you could remove them to make it more uniform.

Reply:
How do I see the line endings?

Reply:
In GitHub they are displayed in red, otherwise you can use the search functionality of your editor in regex mode and search for `\r`.